packages/bp/src/core/app/botpress.ts

```diff
@@ -203,6 +203,7 @@ export class Botpress {
       } else {
         const { endpoint } = config.nluServer
         this.logger.info(`NLU server manually handled at: ${endpoint}`)
+        process.NLU_ENDPOINT = endpoint
       }
```

Comment:
Why does this need to be added here?

Reply:
To be effective the diag must avoid using any of botpress' internals and reflect the true state of the app. The nlu endpoint can come from the module config, or from 2 env variables, so there's no way to be 100% sure of what the effective value is (and I cant use the ghost to read the config in the db)